applications/StructuralMechanicsApplication/tests/test_patch_test_small_strain.py

```diff
@@ -52,10 +52,12 @@ def _apply_material_properties(self,mp,dim):
         g = [0,0,0]
         mp.GetProperties()[1].SetValue(KratosMultiphysics.VOLUME_ACCELERATION,g)
 
-        if(dim == 2):
+        if dim == 2:
             cl = StructuralMechanicsApplication.LinearElasticPlaneStress2DLaw()
-        else:
+        elif dim == 3:
             cl = StructuralMechanicsApplication.LinearElastic3DLaw()
+        else:
+            cl = StructuralMechanicsApplication.LinearElasticAxisym2DLaw()
```

Comment:
then we are not going to enter here right?

Reply:
Wait...